src/source/PeerConnection/SessionDescription.c

```diff
@@ -795,11 +795,12 @@ STATUS populateSingleMediaSection(PKvsPeerConnection pKvsPeerConnection, PKvsRtp
             attributeCount++;
         }
     } else if (pRtcMediaStreamTrack->codec == RTC_CODEC_UNKNOWN) {
-        CHK_STATUS(hashTableGet(pUnknownCodecRtpmapTable, unknownCodecHashTableKey, (PUINT64) &rtpMapValue));
+        CHK_STATUS(hashTableGet(pUnknownCodecRtpmapTable, unknownCodecHashTableKey, &rtpMapValue));
```

Comment:
Not fully understanding this. If a char* and a UINT64* are both the same size of 32 bits on 32-bit systems, and a char is still just 1 byte on both 32 and 64 bit systems, then why would we expect different behavior between the two systems?

Reply:
What is stated there is correct.  While a char** is what is passed to the function, inside `hashTableGet` we dereferenced it before setting it (char** and char* are the same size, they are both pointers). The value that's assigned is a char* and it's assigned a UINT64 which is the bug, but since it was cast to a PUINT64 we will attempt (and succeed) to write the full 8 bytes, which will overwrite the half of `payloadType` and produce the garbage value.

Reply:
I agree with that, but isn't it that the only casting being done is to a char** and the char* is just assigned a value?